vector/src/withoutvoip/java/im/vector/app/features/call/webrtc/WebRtcCall.kt

```diff
@@ -115,7 +116,9 @@ class WebRtcCall(
 
     override fun onStateUpdate(call: MxCall) = Unit
 
-    fun startSharingScreen() = Unit
+    fun startSharingScreen(videoCapturer: VideoCapturer) = Unit
 
     fun stopSharingScreen() = Unit
+
+    fun isSharingScreen(): Boolean =  false
```

Comment:
Did you check all voip variants ?

Reply:
i test with voip variant and it's working :)